supplementary_style_guide/style_guidelines/specific-doc-types.adoc

```diff
@@ -2,13 +2,44 @@
 [[specific-doc-types]]
 = Specific documentation types
 
+[[prerequisites]]
+== Prerequisites
+
+When writing prerequisites, be as clear and concise as possible. You can use the passive voice, _if necessary_, to achieve that end.
+
+Write prerequisites as checks that are true or that the user must have completed before they begin a procedure. They can be actions that the user, another person, or piece of technology has completed. Prerequisites can also include items that the user must have ready before beginning the procedure.
+
+* The passive voice might be appropriate for a prerequisite that is not completed by the current user. For example, having a configuration enabled by a system admin.
+
+* Avoid using imperative formations.
+
+* Use parallel language when writing prerequisites. For example, if one bullet is a complete sentence, the other bullets should be complete sentences. But one bullet could be passive voice and another active voice.
```

Comment:
Hi @stoobie This couple of sentences could use a tiny bit tightening up to be IBM style compliant. Suggest:

Use parallel language when you write prerequisites. For example, if one bullet is a complete sentence, write the other bullets as complete sentences. But one bullet can be passive voice and another active voice.

This suggestion removes the we-should-avoid 'could' and 'should' and makes it more user action oriented. 

everything else looks great!